app/models/news.rb

```diff
@@ -7,6 +7,7 @@ class News < ApplicationRecord
 
   scope :visible, -> { where(is_visible: true) }
   scope :ordered, -> { order('created_at DESC') }
+  scope :osbb_by_user, ->(user_id) { where(osbb_id: user_id.osbb_id) }
```

Comment:
in controller you pass current_user as an argument, `News.osbb_by_user(current_user).visible`, so better to name argument user, instead `user_id`